components/board/fake/board.go

```diff
@@ -420,7 +422,7 @@ func (s *DigitalInterruptWrapper) reset(conf board.DigitalInterruptConfig) error
 func (s *DigitalInterruptWrapper) Value(ctx context.Context, extra map[string]interface{}) (int64, error) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
-	return s.di.Value(ctx, extra)
+	return rand.Int63n(100), nil
```

Comment:
couldn't see a way to set the value as clearly for digital interrupts, but I'm going to continue looking into it

Reply:
The value of the interrupt is the number of times it has ticked since it was created. The value should never decrease. I think we should still return `s.di.Value()`, but before then you might call `s.di.Tick` a (small) random number of times to increase the value. ...but that might be annoying, because the `nanoseconds` argument should also never decrease (it's nanoseconds since the board booted up, IIRC), and ideally `high` would alternate true-false-true-false.

...if the `DigitalInterruptWrapper` stored an int64 named `currentTime`, we could do something like this:
```go
func (s *DigitalInterruptWrapper) Value(ctx context.Context, extra map[string]interface{}) (int64, error) {
	s.mu.Lock()
	defer s.mu.Unlock()
	s.currentTime += 1
	s.di.Tick(ctx, true, s.currentTime)
	s.currentTime += 1
	s.di.Tick(ctx, false, s.currentTime)
	return s.di.Value(ctx, extra)
}
```
Would that work? It doesn't have the randomness of this PR, but it at least changes the value.

Reply:
...or even put those middle 4 lines in a for loop that iterates `rand.Int(5)` times! and that brings back the randomness.

Reply:
I guess I'm wondering how important it is that this behaves like an actual digital interrupt. I assumed it didn't matter if it decreased because it's all fake anyways? but I can test your suggestion if you think it's valuable for the behavior to be at least somewhat similar to a real interrupt

Reply:
We discussed offline: there is benefit in having the fake board act like a real one, to set users' expectations correctly. There is also benefit in not working too hard on simulating a real board, because it's not a very important thing to do. So, the plan is to try having real-like behavior unless it's a hassle.

Reply:
I've tried a couple variations of this but it's causing issues. no error logs, but the board doesn't show up on the RC card and then the server hangs on shut down.